starlite/template/__init__.py

```diff
@@ -1,3 +1,38 @@
+from typing import Any
+from warnings import warn
+
 from .base import TemplateEngineProtocol, TemplateProtocol
 
 __all__ = ("TemplateEngineProtocol", "TemplateProtocol")
+
+
+def __getattr__(name: str) -> Any:
+    """Provide lazy importing as per https://peps.python.org/pep-0562/"""
+
+    if name not in {"JinjaTemplateEngine", "MakoTemplateEngine", "MakoTemplate"}:
+        raise AttributeError(f"Module {__package__} has no attribute {name}")
+
+    if name == "JinjaTemplateEngine":
+        from starlite.contrib.jinja import JinjaTemplateEngine
+
+        export: Any = JinjaTemplateEngine
+        module = "jinja"
+    elif name == "MakoTemplateEngine":
+        from starlite.contrib.mako import MakoTemplateEngine
+
+        export = MakoTemplateEngine
+        module = "mako"
+    else:
+        from starlite.contrib.mako import MakoTemplate
```

Comment:
I don't understand this part. If the imported name is anything besides `JinjaTemplateEngine` and `MakoTemplateEngine` it will default to `MakoTemplate`? Why?

Reply:
It raises an exception a few lines above if the name is not in these three

Reply:
Yeah.. You're right. I guess I'm not fully awake yet. Nevermind then